components/03-modules/active-filters/_active-filters.scss

```diff
@@ -0,0 +1,74 @@
+$active-filters__close-size          : 48px !default;
+$active-filters__remove-padding      : 0 !default;
+$active-filters__icon-fill           : $gray !default;
+$active-filters__icon-size           : $font-size-base !default;
+$active-filters__label-font-weight   : $font-weight-bold !default;
+$active-filters__clear-border        : 2px solid $red !default;
+$active-filters__clear-height        : 48px !default;
+$active-filters__clear-font-size     : 14px !default;
```

Comment:
shouldn't we use here `$font-size-base` ?

Reply:
Done, thanks ;)